kinde_sdk/auth/route_middleware.py

```
@@ -0,0 +1,375 @@
+                    loop = asyncio.get_running_loop()
+                    if loop.is_running():
+                        self._logger.warning(
+                            "Unexpected: running event loop detected in Flask middleware. "
+                            "This may cause issues with route protection."
+                        )
+                except RuntimeError:
+                    # No running loop, which is expected for Flask
+                    pass
+                    
+                return asyncio.run(self.validate_request_access(flask_request))
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)